src/components/FlowRunsAccordion.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <template v-if="flowIds">
+    <p-accordion :sections="flowIds" class="flow-runs-accordion">
+      <template #header="{ section: flowId, id, toggle, content, selected }">
+        <FlowRunsAccordionHeader :flow="getFlow(flowId)!" v-bind="{ id, content, toggle, filter, selected }" />
+      </template>
+      <template #content="{ section: flowId }">
+        <FlowRunsAccordionContent :flow-id="flowId" :filter="filter" />
+      </template>
+    </p-accordion>
+  </template>
+</template>
+
+<script lang="ts" setup>
+  import { computed } from 'vue'
+  import FlowRunsAccordionContent from '@/components/FlowRunsAccordionContent.vue'
+  import FlowRunsAccordionHeader from '@/components/FlowRunsAccordionHeader.vue'
+  import { useFlows } from '@/compositions/useFlows'
+  import { FlowRunsFilter, FlowsFilter } from '@/models/Filters'
+  import { Flow } from '@/models/Flow'
+  import { toMap } from '@/utilities'
+
+  const props = defineProps<{
+    filter: FlowRunsFilter,
+  }>()
+
+  const flowsFilter = computed<FlowsFilter>(() => {
+    // eslint-disable-next-line no-unused-vars
+    const { sort, limit, offset, ...filter } = props.filter
```

Comment:
This is definitely a way to accomplish this but can't we instead `Omit` the fields we don't want callers to pass? That would give a much clearer expectation of what this component can and can't accept without needing to bypass the linter and create an intermediary computed

Reply:
You can Brandon commented on this which makes me second guess this approach. But basically this accordion groups flow runs by their flow. So this component accepts a `FlowRunsFilter` as a prop but then here it converts it into a `FlowsFilter` so that we can grab all the flows for the flow runs we need to display. 

So the caller can pass these things to restrict how many flow runs they want to display. But we need to omit them when getting flows. Maybe there is a clearer/better way to do so.

Reply:
Since the `flowId` exists on every flow run, could we do something like: `new Set(flowRuns.map(run => run.flowId)` and make the header responsible for fetching that flow with `useFlow(id)`? Then you could pass the filter through without modifying it since it's still just for flow runs

Reply:
Hmm that's clever. But then I'd have to fetch all the flow runs, which I'm currently not doing. Only the expanded flow fetches its flow runs.